### ISSUE TITLE
Changing iOS Prefix

### ISSUE THREAD

**Issue by jguerinet** [JetBrains/kotlin-native#1479]
Hi, 

I am currently outputting an iOS framework from a multiplatform project. It seems that the framework name is also used as the prefix for all of the generated files (so if I name the framework `Thing`, all of the outputted files are `ThingModel`, `ThingUtil`). Is there a way to change this prefix, like make it just the first letter of the artifact name?
In the calculator example this seems to be happening: for example [here](https://github.com/JetBrains/kotlin-native/blob/master/samples/calculator/ios/calculator/ViewController.swift) the module is named `KotlinArithmeticParser` but the class is called `KAPPartialParser`. I went through the build files and found where `KotlinArithmeticParser` was being set ([here](https://github.com/JetBrains/kotlin-native/blob/master/samples/calculator/ios/build.gradle)), but not `KAP`. Is it done automatically because it has more than 1 word? 

Thanks

**Comment by olonho** [comment]
Have you tried solution suggested in https://github.com/JetBrains/kotlin-native/issues/1396#issuecomment-372357214?

**Comment by StefMa** [comment]
I think it is worth to add it to the documentation, no?
https://github.com/JetBrains/kotlin-native/blob/master/GRADLE_PLUGIN.md

**Comment by jguerinet** [comment]
Yes @olonho, that works great. I'm sorry, my search of previous issues was obviously not thorough enough. 
I'm happy to close this issue or keep it open and rename it for the documentation piece that @StefMa mentioned.
